src/python_bring_api/bring.py

```diff
@@ -713,3 +736,155 @@ async def notifyAsync(self, listUuid: str, notificationType: BringNotificationTy
+                r.raise_for_status()
+            
+            try:
+                return await r.json()
+            except JSONDecodeError as e:
+                _LOGGER.error(f'Exception: Cannot get user settings for uuid {self.uuid}:\n{traceback.format_exc()}')
+                raise BringParseException('Loading user settings failed during parsing of request response.') from e
+        except asyncio.TimeoutError as e:
+            _LOGGER.error(f'Exception: Cannot get user settings for uuid {self.uuid}:\n{traceback.format_exc()}')
+            raise BringRequestException('Loading user settings failed due to connection timeout.') from e
+        except aiohttp.ClientError as e:
+            _LOGGER.error(f'Exception: Cannot get user settings for uuid {self.uuid}:\n{traceback.format_exc()}')
+            raise BringRequestException('Loading user settings failed due to request exception.') from e
+
+
+    def getArticleTranslations(self, locale: str, invert: bool = False) -> Dict:
+        """
+        Get articles translation table.
+        
+        Parameters
+        ----------
+        locale : str
+           locale of the translation table.
+        invert : str
+            Return the translation table inverted.
+        
+       Returns
+        -------
+        dict
+            The JSON response as a dict.
+        
+
+        Raises
+        ------
+        BringRequestException
+            If the request fails.
+        BringParseException
+            If the parsing of the request response fails.
+       """ 
+        async def _async():
+            async with aiohttp.ClientSession() as session:
+                self._session = session
+                res = await self.getArticleTranslationsAsync(locale, invert)
+                self._session = None
+                return res
+        return asyncio.run(_async())
+
+    async def getArticleTranslationsAsync(self, locale: str, invert: bool = False) -> Dict:
+        """
+        Get articles translation table.
+        
+        Parameters
+        ----------
+        locale : str
+           locale of the translation table.
+        invert : str
+            Return the translation table inverted.
+        
+       Returns
+        -------
+        dict
+            The JSON response as a dict.
+        
+
+        Raises
+        ------
+        BringRequestException
+            If the request fails.
+        BringParseException
+            If the parsing of the request response fails.
+       """ 
+        _supported_locales = {
+            'en-AU', 'de-DE', 'fr-FR', 'it-IT','en-CA', 'nl-NL', 
+            'nb-NO','pl-PL', 'pt-BR', 'ru-RU', 'sv-SE', 'de-CH', 
+            'fr-CH', 'it-CH', 'es-ES', 'tr-TR', 'en-GB', 'en-US'
+            'hu-HU', 'de-AT'
+            }
+        if locale not in _supported_locales:
+            _LOGGER.debug('Locale %s not supported by Bring. Ommitting translation.', locale)
+            return {}

```

Comment:
This is an error imo, and should raise.

Reply:
Might even have to be factored out as a "check" and called from each function to verify that the local provided is valid (or `None`).

Reply:
or should we just not check and if the request returns a 404 we know anyway that the language is not supported.

Reply:
Might be a good idea as well!

Reply:
The webserver does not return a 404, it returns a 200 and loads the webApp which results in a JSONDecoderError 🙄

Reply:
Haha, wtf